docs/src/languages/php/composer-auth.md

```diff
@@ -1,9 +1,10 @@
 ---
 title: "Authenticated Composer repositories"
-
 sidebarTitle: "Authenticated Composer"
 description: |
   Some PHP projects may need to use a private, third party Composer repository in addition to the public Packagist.org repository.  Often, such third party repositories require authentication in order to download packages, and not everyone is comfortable putting those credentials into their Git repository source code (for obvious reasons).
```

Comment:
Definitely out of scope, but we can change it (with slightly different punctuation, especially since you've pointed against using parentheses before :wink: ).